recengine/data_parser.py

```diff
@@ -144,8 +142,8 @@ def ttrdata_from_csv_population_4_weeks(logs_path, individuals_path):
     data = pd.DataFrame(columns=headers)
 
     # Transform data
-    for index, ind in individuals.iterrows():
-        p_id = str(ind.get("ID"))
+    for ID in ttr_data:
+        p_id = str(ID)
 
         ttr = ttr_data.get(p_id)
```

Comment:
Är inte super på python men loopen borde kunna skrivas for ID, ttr in ttr_data.items(): tror jag så slipper man markerade rader.

Borde inte heller behöva konvertera ID till sträng då den redan ska vara det.

Ovan är inget som behöver ändras dock om du inte känner för det! Blir lite mer koncist och lättläst.

Reply:
Indeed, ibland går det fort när jag fipplar :P